### PR TITLE
Add optional pull-cert-manager-e2e-openshift-v3-11

### PR DESCRIPTION
**Pull request by meyskens**
Add the OpenShift v3.11 e2e test currently as an optional test

### CODE REVIEW

**Comment by meyskens** [comment]
seems like my git setup didn't have this upstream.... will rebase

File: config/jobs/cert-manager/cert-manager-presubmits.yaml [hunk 1]
```diff
@@ -498,3 +498,61 @@ presubmits:
...
+    cluster: gke
+    context: pull-cert-manager-e2e-openshift-v3-11
+    always_run: false
+    optional: true
+    max_concurrency: 4
+    agent: kubernetes
+    decorate: true
+    branches:
+    - master
+    - release-0.14
```
Comment (munnerz) [inline config/jobs/cert-manager/cert-manager-presubmits.yaml:512]: I think this shouldn't run against `release-0.14`? All of the jobs in this file should be against `release-0.15` and master (we have a dedicated directory for 'previous' jobs already). Looks like we may have forgotten to update it!

Comment (meyskens) [reply config/jobs/cert-manager/cert-manager-presubmits.yaml]: @munnerz my local branch was like 2 months behind or something... fixed in the rebase

**Pull request merged**